featuretools/tests/primitive_tests/test_datetoholiday_primitives.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from featuretools.primitives.standard.transform_primitive import DateToHoliday
+
+
+def test_datetoholiday():
+    date_to_holiday = DateToHoliday()
+
+    dates = pd.Series([
+        datetime(2016, 1, 1),
+        datetime(2016, 2, 27),
+        datetime(2017, 5, 29, 10, 30, 5),
+        datetime(2018, 7, 4)])
+
+    holiday_series = date_to_holiday(dates).tolist()
+
+    assert holiday_series[0] == "New Year's Day"
+    assert np.isnan(holiday_series[1])
+    assert holiday_series[2] == "Memorial Day"
+    assert holiday_series[3] == "Independence Day"
```

Comment:
Do we want to add some additional tests?

```
    def test_nat(self):
        primitive_instance = self.primitive()
        primitive_func = primitive_instance.get_function()
        case = pd.Series([
            '2019-10-14',
            'NaT',
            '2016-02-15',
            'NaT',
        ]).astype('datetime64')
        answer = ['Columbus Day', np.nan, "Washington's Birthday", np.nan]
        given_answer = primitive_func(case).astype('str')
        np.testing.assert_array_equal(given_answer, answer)

    def test_valid_country(self):
        primitive_instance = self.primitive(country='Canada')
        primitive_func = primitive_instance.get_function()
        case = pd.Series([
            '2016-07-01',
            '2016-11-11',
            '2017-12-26',
            '2018-09-03',
        ]).astype('datetime64')
        answer = ['Canada Day', np.nan, 'Boxing Day', 'Labour Day']
        given_answer = primitive_func(case).astype('str')
        np.testing.assert_array_equal(given_answer, answer)

    def test_multiple_countries(self):
        primitive_mexico = self.primitive(country='Mexico')
        primitive_func = primitive_mexico.get_function()
        case = pd.Series([datetime(2000, 9, 16),
                          datetime(2005, 1, 1)])
        assert len(primitive_func(case)) > 1
        primitive_india = self.primitive(country='IND')
        primitive_func = primitive_mexico.get_function()
        case = pd.Series([datetime(2048, 1, 1),
                          datetime(2048, 10, 2)])
        primitive_func = primitive_india.get_function()
        assert len(primitive_func(case)) > 1
        primitive_uk = self.primitive(country='UK')
        primitive_func = primitive_uk.get_function()
        case = pd.Series([datetime(2048, 3, 17),
                          datetime(2048, 4, 6)])
        assert len(primitive_func(case)) > 1
        countries = ['Argentina', 'AU', 'Austria', 'BY', 'Belgium',
                     'Brazil', 'Canada', 'Colombia', 'Croatia',
                     'England', 'Finland', 'FRA', 'Germany',
                     'Germany', 'Italy', 'NewZealand', 'PortugalExt',
                     'PTE', 'Spain', 'ES', 'Switzerland', 'UnitedStates',
                     'US', 'UK', 'UA', 'CH', 'SE', 'ZA']
        for x in countries:
            self.primitive(country=x)
```

Reply:
yes, thank you